packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -10,7 +10,30 @@ const componentSelector = 'pie-cookie-banner';
 
 export class PieCookieBanner extends LitElement implements CookieBannerProps {
     render () {
-        return html`<h1 data-test-id="pie-cookie-banner">Hello world!</h1>`;
+        return html`
+        <aside data-test-id="pie-cookie-banner" class="c-cookieBanner">
+            <h2 class="c-cookieBanner-title">Cookies</h2>
+            <div class="c-cookieBanner-body">
+                <p>We use our own and third party cookies and other tech to enhance and personalise your user experience,
+                optimize analytics, and show ads with third parties
+                (read our <pie-link variant="inverse">Statement</pie-link>).
+                Necessary cookies are always set. Click <pie-link variant="inverse">Necessary only</pie-link>
+                to continue without accepting more. Click <pie-link variant="inverse">Manage preferences</pie-link>
+                to share your preferences or <pie-link variant="inverse">Accept all</pie-link>.</p>
+            </div>
+
+            <div class="c-cookieBanner-actions">
+                <pie-button variant="primary" isFullWidth="true" size="small-expressive">
+                    Accept all
+                </pie-button>
+                <pie-button variant="outline" isFullWidth="true" size="small-expressive">
+                    Necessary only
+                </pie-button>
+                <pie-link variant="inverse" size="medium" isBold="true">
```

Comment:
Super minor (and happy to part of a subsequent PR), but don't need the size prop here – `medium` is default.

Reply:
cheers - will fix forward!